test/games/strategy/triplea/delegate/MoveDelegateTest.java

```diff
@@ -1047,8 +1047,8 @@ public void testReloadTransportAfterRetreatAllied() {
     retreatingLandUnits.addAll(karelia.getUnits().getMatches(Matches.isUnitAllied(russians, m_data)));
     final Integer retreatingLandSizeInt = retreatingLandUnits.size();
     // Get the defending land units that and their number
-    final List<Unit> defendingLandUnits = new ArrayList<>();
     retreatingLandUnits.addAll(karelia.getUnits().getMatches(Matches.isUnitAllied(british, m_data)));
+    final List<Unit> defendingLandUnits = new ArrayList<Unit>();
```

Comment:
Why not spend a few minutes while we are looking at this code to make a 1 line commit to fix the merge? This will probably never get fixed if we leave it now since you've already mass moved to diamond notation.